libraries/AP_HAL_ChibiOS/hwdef/NxtPX4v2/README.md

```diff
@@ -0,0 +1,92 @@
+# NxtPX4v2 Flight Controller
+
+The NxtPX4v2 is an open-source hardware designed and maintened by [HKUST UAV-Group](https://github.com/HKUST-Aerial-Robotics/Nxt-FC). And it is produced by [MicoAir Tech.](http://micoair.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - BMI088+BMI088(DUAL) IMU
+ - SPL06 barometer
+ - 12V 2.5A BEC; 5V 2.5A BEC
+ - SDCard
+- 7x UART
+- 8x PWM
+- 1x I2C
+- 1x SPI
+- 1x SWD
+
+## UART Mapping
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART2 (DMA-enabled)
+ - SERIAL2 -> UART4 (DMA-enabled)
+ - SERIAL3 -> UART1 (GPS, DMA-enabled)
+ - SERIAL4 -> UART3 (VTX-HD, DMA-enabled)
+ - SERIAL5 -> UART7 (ESC Telemetry, DMA-enabled)
+ - SERIAL6 -> UART5 (RCIN, DMA-enabled)
+ - SERIAL7 -> UART8 (DMA-enabled)
+
+## RC Input
+
+The default RC input is configured on the UART5_RX inverted from the SBUS pin. Other RC  protocols  should be applied at other UART port such as UART2, UART4 or UART8, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL6 _Protocol to something other than '23'
```

Comment:
```suggestion
The default RC input is configured on the UART5. The SBUS pin is inverted and connected to RX5. Non SBUS,  single wire serial inputs can be directly tied to RX5 if SBUS pin is left unconnected. RC could  be applied instead at a different UART port such as UART2, UART4 or UART8, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL6 _Protocol to something other than '23'
```
this assumes that the SBUS inverter does not interfere with RX5 if unconnected. This is the normal configuration for most boards. If not, I can propose a different wording.

Reply:
Fixed. Thanks!